examples/plot_fmri_data_example.py

```diff
@@ -189,8 +196,11 @@ def preprocess_haxby(subject=2, memory=None):
 #############################################################################
 # Now, the clustered inference algorithm which combines parcellation
 # and high-dimensional inference (c.f. References).
-beta_hat, pval_cdl, _, one_minus_pval_cdl, _ = clustered_inference(
-    X, y, ward, n_clusters
+ward_, beta_hat, theta_hat, omega_diag = clustered_inference(
+    X, y, ward, n_clusters, scaler_sampling=StandardScaler()
+)
+beta_hat, pval_cdl, _, one_minus_pval_cdl, _ = clustered_inference_pvalue(
```

Comment:
To me, this function division into two sequential functions means that we do not rely on the right concepts. We need to have a serious discussion on  this API choice.

Reply:
Do you want to have it before or after the Sprint?

Reply:
see issue #201, on this topic.